### ISSUE TITLE
Not available on npm registry yet?

### ISSUE THREAD

**Issue by KingScooty** [luin/ioredis#10]
Getting the following error when trying to install.

```
$ npm install ioredis --save
npm ERR! fetch failed https://registry.npmjs.org/ioredis/-/ioredis-1.0.5.tgz
npm WARN retry will retry, error on last attempt: Error: fetch failed with status code 404
```

**Comment by KingScooty** [comment]
Installing fine now. Must have been some server weirdness.

**Comment by luin** [comment]
:)
I have just published v1.0.6. It seems to be a bug of npm.

**Comment by KingScooty** [comment]
:)
